Examples/Io/Csv/include/ActsExamples/Io/Csv/CsvTrackingGeometryWriter.hpp

```diff
@@ -40,6 +40,12 @@ class CsvTrackingGeometryWriter : public IWriter {
     std::string outputDir;
     /// Number of decimal digits for floating point precision in output.
     std::size_t outputPrecision = std::numeric_limits<float>::max_digits10;
+    /// Write sensitive surfaces
+    bool writeSensitives = true;
+    /// Write boundary surfaces
+    bool writeBoundaries = false;
+    /// Split boundary surfaces
+    bool splitBoundaries = false;
```

Comment:
Using the plural here sounds weird to me. I think we were using singular, i.e. `writeSensitive`, for most other options. I have no good reason for either, just that `writeSensitives` really sounds strange to me.

Reply:
ok, no worries, can change that.